app/app/settings.py

```diff
@@ -200,6 +200,10 @@
 SECURE_HSTS_PRELOAD = True
 SECURE_HSTS_SECONDS = 3600
 
+# List of github usernames to not count as comments on an issue
+IGNORE_COMMENTS_FROM = ['gitcoinbot', ]
```

Comment:
put this here to make it easy to override

Reply:
this is very nice.. thanks